File: 백준/Bronze/14626. ISBN/ISBN.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws Exception {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String isbn = br.readLine();
        int result = 0;
        boolean isEven = false;

        for (int i = 0; i < 13; i++) {
            char ch = isbn.charAt(i);
            if (ch == '*') {
                if (i % 2 != 0) {
                    isEven = true;
                }
                continue;
            }
            int num = ch - '0';
            result += num * (i % 2 == 0 ? 1 : 3);
        }

        if (isEven) {
            for (int i = 0; i < 10; i++) {
                if ((result + (i * 3)) % 10 == 0) {
                    System.out.println(i);
                    break;
                }
            }
        } else {
            System.out.println((10 - result % 10) % 10);
        }
    }
}

File: 백준/Bronze/14626. ISBN/ISBN.py
isbn = input()
result = 0
is_even = False
for i in range(13):
    if isbn[i] == '*':
        if i % 2 != 0:
            is_even = True
        continue
    result += int(isbn[i]) * (1 if i % 2 == 0 else 3)

if is_even == True:
    for i in range(10):
        if (result + (i * 3)) % 10 == 0:
            print(i)
            break
else:
    print(10 - result % 10)